docs/migration/manual-migration.mdx

```diff
@@ -13,69 +13,95 @@ parts, or debug issues in the migration CLI output.
+In version 2, the second parameter of `makeCommandDispatcherWithCommandFeedback`
+was an optional boolean (default `true`) and if this parameter was evaluated to
+a truthy value, the resulting component had the responsibility of displaying the
 result of the command with the `InlineAlert` component. In version 3,
 `makeCommandDispatcherWithCommandFeedback` now expects an optional React
 component to display the result of the command.
 
 To cope with this change, you need to review all components using
 `makeCommandDispatcherWithCommandFeedback`:
 
-* if `makeCommandDispatcherWithCommandFeedback` is called with a falsy second
+- if `makeCommandDispatcherWithCommandFeedback` is called with a falsy second
   parameter, you don't have to change anything
-* if `makeCommandDispatcherWithCommandFeedback` is called with a truthy second
+- if `makeCommandDispatcherWithCommandFeedback` is called with a truthy second
   parameter or without the second parameter, you have to adapt your code. To
   have exactly the same behavior as before, the new call should look like:
   ```jsx
-  makeCommandDispatcherWithCommandFeedback({ /* commands definition */ }, ({ success, message }) => {
-    return <InlineAlert appearance={success ? "success" : "error"}>{message}</InlineAlert>;
-  })
+  makeCommandDispatcherWithCommandFeedback(
+    {
+      /* commands definition */
+    },
+    ({ success, message }) => {
+      return (
+        <InlineAlert appearance={success ? "success" : "error"}>
+          {message}
+        </InlineAlert>
+      );
+    }
+  );
   ```
 
 ## Sass Migration
 
-In the latest version of Front-Commerce we converted all global Sass variables to CSS
-variables during the [automated migration](./automated-migration). This allows
-for a more modular approach to theming and a better developer experience.
+In the latest version of Front-Commerce we converted all global Sass variables
+to CSS variables during the [automated migration](./automated-migration). This
+allows for a more modular approach to theming and a better developer experience.
 However, there are still some manual steps that need to be taken to complete the
 migration.
 
-### Media Mixin
+### Breakpoints Removal
 
-If you make use of the
-[`media-breakpoint-up`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L23-34)
-mixin, you will need to directly import the
-[\_breakpoints.scss](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss)
-file at the location where it is used.
+The following scss helpers have been removed. If you were using them, you have
+to manually add them back to your project or update implementation
```

Comment:
> or update implementation

How would an integrator know what they should do? The codemod comment states "Please follow our manual migration guide: https://developers.front-commerce.com/docs/remixed/migration/manual-migration#breakpoints-removal" but the migration guide doesn't bring any value IMO.

**suggestion:** document how the implementation could be updated to not use these removed variables (i.e: do like in our core)

Reply:
I added another section with a more recommended approach, to replace userland code with simpler and direct media queries in d2a22b3c1e7d4631dea4223031ade5045a56246c